docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc

```diff
@@ -42,46 +39,28 @@ The following columns have been added to the corresponding pages:
+=== New Task Inbox and Task Administration pages
+The former Task List page has been divided into the Task Inbox and Task Administration pages. The Task Inbox page is used by task operators to work with their assigned (or potentially assigned) tasks. The Task Administration page is used by administrators to manage tasks assigned to other users. This page is only available for users with `admin` or `process-admin` roles. It is similar to the *Admin filter* tab on the former Task List page.
 
-=== Error Handling
-While running a process instance or a job, the system now detects and stores execution errors such as exceptions after the maximum number of retries has been exceeded. If an error occurs, the system generates an event that contains the following data:
+=== High availability
+High availability and clustering for {CENTRAL} is now configured with JMS and NFS storage.
 
---
-[cols="1,2",options="header"]
-|===
-| Data
-| Fields
-
-.3+| Process model
-| processModelID
-| processModelVersion
-| processModelName
-
-.2+| Process instance that has failed
-| processInstanceID
-| customProcessID
-
-.5+| Activity that has failed
-| activityID
-| activityName
-| activityType
-| iteration
-| status
-
-.2+| Error data
-| errorMessage: Message with the error that has occurred.
-| errorStack: Stack with error detail.
-|===
---
+=== Enabling the Content Management System (CMS) page
+{PRODUCT} settings allow you to enable the CMS page.
 
-The process instance list includes a new column that displays the number of errors per process instance which are not marked as acknowledged. When you select the column, a window appears which enables you to navigate to the new Error page.
+.Procedure
+. Go to *Settings*:> *Roles*:> *admin*.
+. Expand *Pages* and click the *Read* field to enable *Page Authoring*.
 
-The Execution Errors page lists any errors that occur during process instances or executor service jobs on a {KIE_SERVER}. Use this page to help troubleshoot issues. It is visible to users with the admin and process-admin roles.
+=== Error Handling
+While running a process instance or a job, the system now detects and stores execution errors such as exceptions after the maximum number of retries has been exceeded. 
+
+The process instance list includes a new column that displays the number of errors per process instance which are not marked as acknowledged. 
+
+The Execution Errors page lists any errors that occur during process instances or executor service jobs on a {KIE_SERVER}. Use this page to help troubleshoot issues. It is visible to users with the `admin` and `process-admin` roles.
```

Comment:
Ambiguous phrase. "lists any errors... or executor service jobs" or "lists any errors that occur during... executor service jobs"?